### ISSUE TITLE
Cannot insert NULL into nullable int64 column

### ISSUE THREAD

**Issue by jdiamond** [cloudspannerecosystem/cassandra-to-spanner-proxy#2]
Create a table like this with spanner-cli:

```sql
create table nullable_int64s (
  id string(max),
  num int64,
  last_commit_ts timestamp options (allow_commit_timestamp=true)
) primary key (id);
```

Insert the metadata:

```sql
INSERT INTO TableConfigurations (KeySpaceName, TableName, ColumnName, ColumnType, IsPrimaryKey, PK_Precedence) VALUES ('test', 'nullable_int64s', 'id', 'text', true, 0);
INSERT INTO TableConfigurations (KeySpaceName, TableName, ColumnName, ColumnType, IsPrimaryKey, PK_Precedence) VALUES ('test', 'nullable_int64s', 'num', 'int', false, 0);
```

I can insert both non-null and null values via spanner-cli:

```sql
insert into nullable_int64s (id, num) values ('a', 1);
insert into nullable_int64s (id, num) values ('b', null);
```

Here is an insert without null and another with null which I am executing via cqlsh connected to the proxy:

```cql
insert into test.nullable_int64s (id, num) values ('c', 1);
insert into test.nullable_int64s (id, num) values ('d', null);
```

When I try to execute that last insert, I get this error in cqlsh:

```
InvalidRequest: Error from server: code=2200 [Invalid query] message="strconv.Atoi: parsing "null": invalid syntax"
```

And this error in the proxy:

```
{"level":"error","time":"2024-10-22T23:51:29.694Z","caller":"proxy/proxy.go:1517","msg":"Error occurred at translator","Query":"insert into test.nullable_int64s (id, num) values ('d', null);","error":"strconv.Atoi: parsing \"null\": invalid syntax","stacktrace":"github.com/ollionorg/cassandra-to-spanner-proxy/third_party/datastax/proxy.(*client).handleQuery\n\t/go/src/cassandra-to-spanner-proxy/third_party/datastax/proxy/proxy.go:1517\ngithub.com/ollionorg/cassandra-to-spanner-proxy/third_party/datastax/proxy.(*client).Receive\n\t/go/src/cassandra-to-spanner-proxy/third_party/datastax/proxy/proxy.go:641\ngithub.com/ollionorg/cassandra-to-spanner-proxy/third_party/datastax/proxycore.(*Conn).read\n\t/go/src/cassandra-to-spanner-proxy/third_party/datastax/proxycore/conn.go:114"}
```

**Comment by mayurkale22** [comment]
Fixed via #7

**Comment by mayurkale22** [comment]
Thank you for reporting this issue and providing detailed steps to reproduce it. We have submitted the fix, which should be available in the next release.
